content/francais/python-turtle/activity-9.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Activité 9 - Colorer votre fleur mandala"
+date: 2024-09-17T13:24:17-07:00
+weight: 12
+draft: false
+---
+
+![alt text width="70%"](../media/mandala-blue.png "fleur mandala avec des pétales bleus")
+
+Maintenant que nous avons la forme d'une fleur mandala, il est temps de la colorier ! Nous allons toujours utiliser la méthode `turtle.color()` ; cependant, cette fois, nous allons utiliser une valeur RGB au lieu d'une valeur de couleur en chaîne, afin de produire une plus grande variété de couleurs.
+
+{{% notice tip %}}
+
+## Couleurs RGB (Rouge-Vert-Bleu)
```

Comment:
Maybe adding the english meaning of the acronym "Red, Green, Blue" can be useful, on top of the colour translation in french.